src/components/App/utils.js

```diff
@@ -0,0 +1,120 @@
+// How many seconds before we unload videos
+const SECONDS_BEFORE_UNLOAD = 30;
+
+function playVideo(video) {
+  if (video.api) {
+    return video.api.play();
```

Comment:
I think this is fine as is, but I wonder if the conditionals are necessary, could we just ignore the API prop or do we need to interact with Odyssey videos that way to keep the API in sync with reality?

Reply:
From memory we do want to interact with the Odyssey videos and keep everything in sync globally if they are Odyssey videos.. but I can't remember why. If it still works directly with video.play then maybe it's not needed?

Reply:
I'd suggest we leave it as is for now, but a possible candidate for future simplification.